test/axes/numericAxisTests.ts

```diff
@@ -16,453 +16,335 @@ describe("NumericAxis", () => {
+        const svg = TestMethods.generateSVG();
+        axis.tickLabelPosition(labelPosition);
+        axis.renderTo(svg);
+
+        const tickLabels = axis.content().selectAll(`.${Plottable.Axis.TICK_LABEL_CLASS}`);
+        const tickMarks = axis.content().selectAll(`.${Plottable.Axis.TICK_MARK_CLASS}`);
+        assert.operator(tickLabels.size(), ">=", 2, "at least two tick labels were drawn");
+        assert.strictEqual(tickLabels.size(), tickMarks.size(), "there is one label per mark");
+
+        tickLabels.each(function(d, i) {
+          TestMethods.assertBBoxNonIntersection(d3.select(this), d3.select(tickMarks[0][i]));
+        });
+        svg.remove();
+      });
+    });
   });
 
-  it("uses the supplied Formatter", () => {
-    let SVG_WIDTH = 100;
-    let SVG_HEIGHT = 500;
-    let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-    let scale = new Plottable.Scales.Linear();
-    scale.range([0, SVG_HEIGHT]);
+  describe("drawing tick labels when bottom oriented", () => {
+
+    let svg: d3.Selection<void>;
+
+    beforeEach(() => {
+      svg = TestMethods.generateSVG();
+    });
+
+    it("does not overlap tick labels in a constrained space", () => {
+      let constrainedWidth = 50;
+      let constrainedHeight = 50;
+      svg.attr("width", constrainedWidth);
+      svg.attr("height", constrainedHeight);
+      let scale = new Plottable.Scales.Linear();
+      let numericAxis = new Plottable.Axes.Numeric(scale, "bottom");
+      numericAxis.renderTo(svg);
 
-    let formatter = Plottable.Formatters.fixed(2);
+      let visibleTickLabels = applyVisibleFilter(numericAxis.content()
+        .selectAll(`.${Plottable.Axis.TICK_LABEL_CLASS}`));
 
-    let numericAxis = new Plottable.Axes.Numeric(scale, "left");
-    numericAxis.formatter(formatter);
-    numericAxis.renderTo(svg);
+      visibleTickLabels.each(function(d, i) {
+        let labelRect = this.getBoundingClientRect();
+        visibleTickLabels[0].slice(i + 1).forEach(function(otherVisibleTickLabel, i2) {
+          let labelRect2 = (<Element> otherVisibleTickLabel).getBoundingClientRect();
+          let rectOverlap = Plottable.Utils.DOM.clientRectsOverlap(labelRect, labelRect2);
+          assert.isFalse(rectOverlap, `tick label ${i} does not overlap with tick label ${i2}`);
+        });
+      });
 
-    let tickLabels = (<any> numericAxis)._element.selectAll("." + Plottable.Axis.TICK_LABEL_CLASS);
-    tickLabels.each(function(d: any, i: number) {
-      let labelText = d3.select(this).text();
-      let formattedValue = formatter(d);
-      assert.strictEqual(labelText, formattedValue, "The supplied Formatter was used to format the tick label");
+      svg.remove();
     });
 
-    svg.remove();
-  });
+    it("ensures that long labels are contained", () => {
+      let scale = new Plottable.Scales.Linear().domain([400000000, 500000000]);
+      let axis = new Plottable.Axes.Numeric(scale, "left");
```

Comment:
why "left" isn't this block for bottom oriented axes?
also, maybe more axis declaration up a level and be consistent variable choice (axis vs numericAxis)